activesupport/lib/active_support/file_evented_update_checker.rb

```diff
@@ -0,0 +1,72 @@
+require 'listen'
+
+module ActiveSupport
+  class FileEventedUpdateChecker
+    attr_reader :listener
+    def initialize(files, directories={}, &block)
+      @files = Set.new
```

Comment:
Are suggesting to remove @files variable and make changes in the `files` argument only?